submissions/smillims/oop-exercise/index.js

```diff
@@ -0,0 +1,64 @@
+/* 
+	Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+*/
+
+'use strict';
+
+class Inhabitats {
+	constructor (species, name, surname, gender, legs, hands, saying) {
+		this.species = species;
+		this.name = name;
+		this.surname = surname;
+		this.gender = gender;
+		this.legs = legs;
+		this.hands = hands;
```

Comment:
OOP is not just fancy class syntax.
The beauty of OOP is that objects may not have properties they do not have by natural design. Not zero or `null` or `undefined`, just no property at all.